app/middlewares/errors.js

```diff
@@ -7,7 +7,8 @@ const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
   [errors.DEFAULT_ERROR]: 500,
   [errors.SERVICE_ERROR]: 504,
-  [errors.VALIDATE_ERROR]: 400
+  [errors.VALIDATE_ERROR]: 400,
+  [errors.COMPARE_ERROR]: 401
```

Comment:
A better name for this could be AUTHENTICATION_ERROR. (with `better` I mean it suits better to it meaning)